### ISSUE TITLE
Capacitor is now forcibly installed on starter projects

### ISSUE THREAD

**Issue by ziftytodd** [ionic-team/ionic-cli#4750]
**Description:**
There used to be a prompt to add capacitor to a new project created using the starter templates, but that option no longer is shown and it's added by default to all projects whether you want it or not. For web based projects, Capacitor isn't needed. Sure, it's easy enough to remove it afterwards, but a bit of a pain and all the dependencies are installed too, so really have to delete node_modules and reinstall from scratch to fully remove it. Same behavior using the new wizard on the web too.

**Steps to Reproduce:**
Run ionic start, pick Angular, pick and template, and capacitor is added without any prompt

**My `ionic info`:**
```
Ionic:

   Ionic CLI : 6.17.0

Utility:

   cordova-res                          : 0.15.3
   native-run (update available: 1.4.0) : 1.3.0

System:

   NodeJS : v14.17.4
   npm    : 6.14.14
   OS     : macOS Big Sur

```

**Comment by Marabiks** [comment]
Besides, the capacitor integration at start automatically install its dependencies with "npm" even if the `--no-deps` option is given to the `start` command. This's a problem because if I wanted to use, for instance, "pnpm" package manager (that I do), I would need to delete the "./node_modules" folder to start correctly, making this process unnecesary bandwidth and time-consuming.

**Comment by ziftytodd** [comment]
@Marabiks agreed. this is also my experience. there's no option to prevent capacitor from being included.
